### PR TITLE
Add opentofu notice

### PR DESCRIPTION
**Pull request by MoonMoon1919**
## Description

Adds notice that Terragrunt supports OpenTofu as of TG version v0.52.0.

image

## TODOs

Read the [Gruntwork contribution guidelines](https://gruntwork.notion.site/Gruntwork-Coding-Methodology-02fdcd6e4b004e818553684760bf691e).

- [ ] Update the docs.
- [ ] Run the relevant tests successfully, including pre-commit checks.
- [ ] Ensure any 3rd party code adheres with our [license policy](https://www.notion.so/gruntwork/Gruntwork-licenses-and-open-source-usage-policy-f7dece1f780341c7b69c1763f22b1378) or delete this line if its not applicable.
- [ ] Include release notes. If this PR is backward incompatible, include a migration guide.

## Release Notes (draft)

Added / Removed / Updated [X].

### CODE REVIEW

**Review by oredavids — DISMISSED** [review]

**Review by denis256 — DISMISSED** [review]

**Review by denis256 — APPROVED** [review]

**Review by denis256 — DISMISSED** [review]

**Review by denis256 — APPROVED** [review]

File: docs/_layouts/default.html [hunk 1]
```diff
@@ -4,6 +4,13 @@
         {% include head.html %}
     </head>
     <body{% if page.slug %} class="{{ page.slug }}{% if page.slug != 'index-page' and page.slug !='how-it-works' and page.slug != 'guides' %} sub-page{% endif %}" {% endif %}>
+        {% include banner.html
+            text="✨&nbsp;&nbsp;Terragrunt v0.52.0 added support OpenTofu. To learn more, see the official"
+            link_url="https://github.com/gruntwork-io/terragrunt/releases/tag/v0.52.0"
+            link_text="release notes."
+            target="_blank"
+            has_button=false
+        %}
```
Comment (ebeneliason) [inline docs/_layouts/default.html:13]: I don't feel that strongly about this change, so I submit if for your consideration. It just makes the notice a little more concise. I was torn on using "Learn more" vs. "View release notes" for the link, but I think the clarity of the latter is nice.
```suggestion
        {% include banner.html
            text="✨&nbsp;&nbsp;Terragrunt v0.52.0 now supports OpenTofu."
            link_url="https://github.com/gruntwork-io/terragrunt/releases/tag/v0.52.0"
            link_text="View release notes"
            target="_blank"
            has_button=false
        %}
```

Comment (MoonMoon1919) [reply docs/_layouts/default.html:13]: I think this is better too - I was looking at it and thinking that it reads a little funky, but couldn't come up with better wording. Let's o with yours!

**Comment by ebeneliason** [comment]
Looks great overall. Made a small copy edit suggestion. Is the link text suitably visible against the gray banner? It's hard to tell from the screenshot if the contrast is high enough.

**Review by ebeneliason — APPROVED** [review]
LGTM

**Pull request merged**